Website/src/components/Navbars/HomeNavbar.jsx

```diff
@@ -72,18 +51,26 @@ function HomeNavbar() {
             isOpen={collapseOpen}
             navbar
           >
-            <Nav navbar>
-              <NavItem>
-            
+            <Nav navbar style={{color: '#ffffff', background: '#171b5d', marginTop: '0px', paddingTop: '50px'}}>
+              <NavItem style={{padding: '15px'}}>
+              Link 1 
               </NavItem>
-              <UncontrolledDropdown nav>
-         
-             
-              </UncontrolledDropdown>
-              <NavItem>
-             
+              <NavItem style={{padding: '15px'}}>
+             Link 2 
                
               </NavItem>
+              <NavItem style={{padding: '15px'}}>
+              Link 3 
+              </NavItem>
+              <NavItem style={{padding: '15px'}}>
```

Comment:
Same previous comment. Please move all styles to the tkot.scss file under an appropriate CSS class name, then reference the CSS class name in the component.